src/graph/client/graphClient.ts

```diff
@@ -367,7 +379,13 @@ export class GraphClient {
       if (this._force) {
         this._force.stop();
       }
+      const resultsBackground = document.getElementById("resultsBackground");
+      if (resultsBackground) {
```

Comment:
Here you're not resizing the SVG, you're setting up the initial size of the simulation, which you can go ahead and do based on the current resultsBackground.clientWidth/Height.  No need for file variables.